SQL/New_patches/2018-08-27_add_name_of_mri_config_file_to_config_module.sql

```diff
@@ -0,0 +1,35 @@
+--- INSERT THE 2 NEW CONFIGs INTO ConfigSettings TABLE
+INSERT INTO ConfigSettings
+  (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber)
+  SELECT
+    'MriConfigFile',
+    'Name of the MRI config file (stored in dicom-archive/.loris_mri/',
```

Comment:
This is missing a closing bracket

Reply:
Good catch! Fixed in the next commit. Thanks!